action.yml

```diff
@@ -5,5 +5,15 @@ runs:
   using: 'docker'
   image: 'docker://peterevans/dockerhub-description:2.1.1'
 branding:
-  icon: 'upload'  
+  icon: 'upload'
   color: 'blue'
+inputs:
+  dockerhubUsername:
+    description: Username to login to Docker Hub. Aliases: DOCKER_USERNAME
+    required: false
+  dockerhubPassword:
+    description: Password to login to Docker Hub. Aliases: DOCKER_PASSWORD
+    required: false
+  dockerhubRepository:
+    description: Explicit Docker Hub repository name. Aliases: DOCKER_REPOSITORY, GITHUB_REPOSITORY
+    required: false
```

Comment:
The `inputs` metadata is for detailing inputs that are passed via the `with` property. It doesn't make sense in this case since the action uses the old-style `env` property. I think we should either remove this section and make users refer to the readme, or reference the `env` properties in yaml comments like this:

```yml
name: 'Docker Hub Description'
author: 'Peter Evans'
description: 'An action to update a Docker Hub repository description from README.md'
# env:
#   DOCKERHUB_USERNAME:
#     description: Username to login to Docker Hub. Aliases: DOCKER_USERNAME
#     required: false
#   DOCKERHUB_PASSWORD:
#     description: Password to login to Docker Hub. Aliases: DOCKER_PASSWORD
#     required: false
#   DOCKERHUB_REPOSITORY:
#     description: Explicit Docker Hub repository name. Aliases: DOCKER_REPOSITORY, GITHUB_REPOSITORY
#     required: false
#   README_FILEPATH:
#     description: Path to the repository readme.
#     default: ./README.md
runs:
  using: 'docker'
  image: 'docker://peterevans/dockerhub-description:2.1.1'
branding:
  icon: 'upload'
  color: 'blue'
```

Reply:
If you make a change suggestion with your reference, I'm happy to accept it in so we can get merged. I was under the impression that `inputs` was also made available over the environment keyed by their YAML key - It would've worked as a side-effect of that, but you're correct that it's a misuse of the features intended functionality.

Reply:
I see what you were trying to do now, but I don't think it would have worked. They would become environment variables prefixed with `INPUT_`. For example, `dockerhubUsername` would become `INPUT_DOCKERHUBUSERNAME`.
ref: https://help.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#example

Reply:
Ah, gotcha - Yeah, glossed over that bit, but the goal was just to get more UI features - Oh well either way. One second and I'll add your change.

Reply:
One other small thing. Let's remove the `required: false` on my suggestion above if you are going to commit that.
```
# env:
#   DOCKERHUB_USERNAME:
#     description: Username to login to Docker Hub. Aliases: DOCKER_USERNAME
#   DOCKERHUB_PASSWORD:
#     description: Password to login to Docker Hub. Aliases: DOCKER_PASSWORD
#   DOCKERHUB_REPOSITORY:
#     description: Explicit Docker Hub repository name. Aliases: DOCKER_REPOSITORY, GITHUB_REPOSITORY
#   README_FILEPATH:
#     description: Path to the repository readme.
#     default: ./README.md
```